host.go

```diff
@@ -30,14 +32,9 @@ type gpbftRunner struct {
 
 	runningCtx context.Context
 	log        Logger
-
-	host *gpbftHost
 }
 
-type gpbftHost struct {
-	Client
-	runner *gpbftRunner
-}
+type gpbftHost gpbftRunner
```

Comment:
Why the type alias?

Reply:
To separate internal (towards gpbft.Participant) and external (towards f3.F3) APIs